Tests/Common/Securities/OptionMarginBuyingPowerModelTests.cs

```diff
@@ -392,6 +392,54 @@ public void TestShortPutMovingFarITM()
             Assert.AreEqual(8936, (double)buyingPowerModel.GetMaintenanceMargin(optionPut), 0.01);
         }
 
+        // ITM
+        [TestCase(OptionRight.Call, 300, 115.75, 415, 19800)] // IB: 19837
+        // OTM
+        [TestCase(OptionRight.Put, 300, 0.45, 415, 3000)] // IB: 3044
+        // ITM
+        [TestCase(OptionRight.Call, 390, 27.5, 415, 11000)] // IB: 11022
+        // OTM
+        [TestCase(OptionRight.Put, 390, 1.85, 415, 6000)] // IB: 6042
+        // OTM
+        [TestCase(OptionRight.Call, 430, 0.85, 415, 6800)] // IB: 6803
+        // ITM
+        [TestCase(OptionRight.Put, 430, 16.80, 415, 9900)] // IB: 9929
+        public void ShortOptionsMargin(OptionRight optionRight, decimal strikePrice, decimal optionPrice, decimal underlyingPrice,
+            double expectedUnitMargin)
+        {
+            var tz = TimeZones.NewYork;
+            var equity = new QuantConnect.Securities.Equity.Equity(
```

Comment:
Nice!
nit comment but I think there's room to create a helper method to create the underlying & option since it's repeated in the different unit tests?